sdc/crypto/test/test_jwe_helper.py

```diff
@@ -0,0 +1,173 @@
+import os
```

Comment:
Agreed, I did try but its quite a big job. I'll raise an issue specifically related to unit tests